tests/test_rest_communication.py

```diff
@@ -123,6 +122,16 @@ def test_get_documents_depaginate(self):
                 {'params': {'page': '2', 'max_results': '101'}, 'quiet': False},
                 {'params': {'page': '3', 'max_results': '101'}, 'quiet': False}
             ]
+        docs = [
+            FakeRestResponse(content={'data': ['data%s' % d], '_links': {'next': {'href': 'an_endpoint?max_results=101&page=%s' % d}}})
+            for d in range(2,1200)
+        ]
+        docs.append(FakeRestResponse(content={'data': ['last piece'], '_links': {}}))
+
+        with patch(ppath('_req'), side_effect=docs) as mocked_req:
+            ret = self.comm.get_documents('an_endpoint', all_pages=True, max_results=101)
+            print(ret)
```

Comment:
Can we assert something about the result of `get_documents` instead of printing it?